### PR TITLE
1. Added Images and Inputs

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

File: components/images.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.image-circular {
+  width: 16rem;
+  height: 16rem;
+  border-radius: 50%;
+}
+
+.image-square {
+  border-radius: 0 !important;
```
Comment (Mohasin-Haque) [inline components/images.css:23]: why use important?

File: components/images.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.img-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```
Comment (Mohasin-Haque) [inline components/images.css:4]: you can make a new class as you have used this property again and again

File: components/inputs.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  margin: 1rem 0;
+  border-radius: 0.5rem;
+}
+
+.input-error {
+  color: var(--red);
+}
+
+.input-success {
+  border: 3px solid var(--green);
```
Comment (Mohasin-Haque) [inline components/inputs.css:33]: Use the color name according to property otherwise why not we directly use the name of color.. Think on it

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good to me but needs to work on the comments.!!

File: components/images.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  align-items: center;
+}
+.images-container {
```
Comment (SHIKHASINGH1506) [inline components/images.css:7]: ".image-container" class name would be more readable.

File: components/inputs.css (hunk 3 above)
Comment (SHIKHASINGH1506) [reply components/inputs.css:33]: You could name this color as "success-color"

**Review by SHIKHASINGH1506 — APPROVED** [review]
Just minor changes

**Pull request merged**